src/extension.ts

```diff
@@ -1275,14 +1280,20 @@ async function reportDeleteResult(resourceId: string, shellResult: ShellResult)
     refreshExplorer();
 }
 
-const deleteKubernetes = async (explorerNode?: explorer.ResourceNode) => {
+async function deleteKubernetes(delMode: KubernetesDeleteMode, explorerNode?: explorer.ResourceNode) {
+
+    let delModeString: string = '';
```

Comment:
Minor: I don't think we need the type annotation here - `string` will be inferred I believe.